src/index.js

```diff
@@ -18,8 +18,14 @@ const onCardEdit = (cardId, newCardData) => {
     return card.id === cardId;
   });
 
+  let newCard = newCardData;
```

Comment:
```suggestion
  const newCard = newCardData;
```
Тут const, потому что саму переменную мы не переписываем.